TypeError: Cannot read properties of undefined (reading 'flowElements')

### Describe the Bug

<!-- A clear and concise description of what the bug is. -->

As part of the BPMN co-pilot prototype testing, the AI model returned a valid BPMN file with no process description. 

The auto-layouter failed:
```
Chatbot.js:95 xml 
<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn">
</bpmn:definitions>

Chatbot.js:112 error TypeError: Cannot read properties of undefined (reading 'flowElements')
    at Layouter.createGridLayout (index.js:711:31)
    at Layouter.handlePlane (index.js:700:25)
    at Layouter.layoutProcess (index.js:694:10)
    at async eval (Chatbot.js:102:13)
```

### Steps to Reproduce

<!-- Clear steps that allow us to reproduce the error. -->

1. call the auto-layouter with a valid BPMN file but missing process description

### Expected Behavior

<!-- A clear and concise description of what you expected to happen. -->
I expect the auto-layouter not to fail, and simply return an empty canvas.

### Environment

- Latest version of the [bpmn-co-pilot-prototype](https://github.com/camunda/bpmn-copilot-prototype)

--- a/lib/Layouter.js
+++ b/lib/Layouter.js
@@ -27,8 +27,10 @@ export class Layouter {
 
     const root = this.getProcess();
 
-    this.cleanDi();
-    this.handlePlane(root);
+    if (root) {
+      this.cleanDi();
+      this.handlePlane(root);
+    }
 
     return (await this.moddle.toXML(this.diagram, { format: true })).xml;
   }

--- a/test/fixtures/empty-definition.bpmn
+++ b/test/fixtures/empty-definition.bpmn
@@ -1,0 +1,2 @@
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_13fbzpq" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="1.16.2">
+</bpmn:definitions>

